docs/docs/features/query.md

```diff
@@ -12,9 +12,14 @@ select * from {{ ref("mymodel") }}
 ```
 
 1. **Run:** performs query in the current environment
-2. **Run Diff:** performs the same query in both environments and diffs the results. 
+2. **Run Diff:** performs the same query in both environments and diffs the results.
+3. **Primary key:** select the primary key(s) used to compare the query results.
```

Comment:
`Primary Key` is not a action. So it should not be included in this list.

Reply:
I created a sub-section for the form column.